bukkit/src/main/java/de/maxhenkel/voicechat/config/ServerConfig.java

```diff
@@ -30,79 +30,79 @@ public ServerConfig(ConfigBuilder builder) {
         voiceChatPort = builder
                 .integerEntry("port", 24454, -1, 65535,
                         "The port of the voice chat server",
-                        "Setting this to \"-1\" sets the port to the Minecraft servers port (Not recommended)"
+                        "Setting this to -1 sets the voice chat port to the Minecraft server port (Not recommended)"
                 );
         voiceChatBindAddress = builder
                 .stringEntry("bind_address", "",
-                        "The IP address to bind the voice chat server on",
-                        "Leave empty to use 'server-ip' of server.properties",
+                        "The IP address to bind the voice chat server to",
+                        "Leave blank to use the 'server-ip' property from server.properties",
                         "To bind to the wildcard address, use '*'"
                 );
         voiceChatDistance = builder
                 .doubleEntry("max_voice_distance", 48D, 1D, 1_000_000D,
-                        "The distance to where the voice can be heard"
+                        "The distance to which the voice can be heard"
                 );
         crouchDistanceMultiplier = builder
                 .doubleEntry("crouch_distance_multiplier", 1D, 0.01D, 1D,
-                        "The multiplier the voice distance will be reduced by when sneaking"
+                        "The multiplier of the voice distance when crouching"
                 );
         whisperDistanceMultiplier = builder
                 .doubleEntry("whisper_distance_multiplier", 0.5D, 0.01D, 1D,
-                        "The multiplier the voice distance will be reduced by when whispering"
+                        "The multiplier of the voice distance when whispering"
                 );
         voiceChatCodec = builder
                 .enumEntry("codec", Codec.VOIP,
-                        "The opus codec"
+                        "The Opus codec",
+                        "Valid values are 'VOIP', 'AUDIO', and 'RESTRICTED_LOWDELAY'"
                 );
         voiceChatMtuSize = builder
                 .integerEntry("mtu_size", 1024, 256, 10000,
-                        "The maximum size in bytes in a voice packet",
-                        "Set this to a lower value if your voice packets don't arrive"
```

Comment:
Why was this removed?

Reply:
From the description of this pull request:
> 2. Configuration file comment revamp:
>    - It also unifies the comments for various configuration options between different platforms/engines so that everyone is treated equally.

Now that I think about it, I should have done it the other way around. Thanks for pointing that out. I'll add it back to both the ServerConfig classes.